actions/src/search_openstack.py

```diff
@@ -66,7 +77,8 @@ def get_preset(
 
         # search openstack_resource ID presets (works for all openstack objects)
         func = {
-            "id_in": lambda properties_list, criteria_list, sort_by_list, **kwargs: query(
+            "id_in": lambda cloud_account, properties_list, criteria_list, sort_by_list, **kwargs: query(
```

Comment:
Are we sticking with this, or is the the legacy way?

As lambdas and methods are equivalent can we change these to methods and simply pass the method in instead. It's a lot more maintainable and allows us to later wrap each in tests

Reply:
This is the old code.